docs/sources/collect/choose-component.md

```diff
@@ -0,0 +1,72 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/collect/choose-component/
+description: Find out which components are useful for which tasks
+title: Choose an Alloy component
+weight: 100
+---
+
+# Choose an Alloy component
+
+[Components][components] are the building blocks of Alloy, and there is a [large number of them][components-ref].
```

Comment:
```suggestion
# Choose a  {{< param "FULL_PRODUCT_NAME" >}} component

[Components][components] are the building blocks of {{< param "FULL_PRODUCT_NAME" >}}, and there is a [large number of them][components-ref].
```

We can use a shorter menu title, and then use the full product name in the title and H1.